pyproject.toml

```diff
@@ -23,6 +24,12 @@ classifiers = [
   "Programming Language :: Python :: 3.7",
   "Programming Language :: Python :: 3.8",
   "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+  "Topic :: Software Development :: Build Tools",
+  "Topic :: System :: Archiving :: Packaging",
+  "Topic :: System :: Software Distribution",
+  "Topic :: Utilities",
```

Comment:
Eh, this doesn't really fit into this PR. But fine with me if you want to save trees.

Reply:
The PyPy totally does and was the impetous. Pex now has official PyPy support since it covers pypy 2 and 3. I'll break out this and the pypy3 changes to 1 PR. The trees are a big problem I'll continue to think on. There has to be some good way to put backpressure into the system that forces us to account for the damage.